markets/storageadapter/dealpublisher.go

```diff
@@ -94,6 +105,9 @@ type PublishMsgConfig struct {
 	MaxDealsPerMsg uint64
 	// Minimum start epoch buffer to give time for sealing of sector with deal
 	StartEpochSealingBuffer uint64
+	// Enable ExternalDealPublishControl to allow users to choose when to send Publish messages
+	// for each deal
+	ExternalDealPublishControl bool
```

Comment:
Let's also change this from ExternalDealPublishControl to ManualPSD for consistency